formidable/app.py

```diff
@@ -21,3 +23,11 @@ def ready(self):
         """
         from .views import check_callback_configuration
         check_callback_configuration()
+
+        # Deprecation warning for django 1.8 and 1.9
+        if django.VERSION[0] is 1:
+            minorVersion = django.VERSION[1]
+            if minorVersion is 8 or minorVersion is 9:
+                warnings.warn("Support for django 1. " + str(minorVersion) +
```

Comment:
I wouldn't do a dynamic message here.
I'd write:

> Support for Django 1.8 and 1.9 is about to be dropped. Please update to a newer version.

Reply:
I've updated the code according to your feedbacks!